modules/library/src/main/scala/zio/elasticsearch/query/InnerHits.scala

```diff
@@ -135,12 +135,10 @@ final case class InnerHits private[elasticsearch] (
   private[elasticsearch] def toStringJsonPair(fieldPath: Option[String]): (String, Json) = {
     val sourceJson: Option[Json] =
       (included, excluded) match {
-        case (Chunk(), Chunk()) =>
-          None
-        case (included, excluded) =>
-          val includes = if (included.isEmpty) Obj() else Obj("includes" -> Arr(included.map(_.toJson)))
-          val excludes = if (excluded.isEmpty) Obj() else Obj("excludes" -> Arr(excluded.map(_.toJson)))
-          Some(includes merge excludes)
+        case (Chunk(), Chunk()) => None
```

Comment:
I guess we can put `None` in the new line, just to be consistent.

Reply:
We are. Everything is inlined here.